docs/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# How Has This To Be Tested?
+
+Please describe the tests (Instructions) to verify the changes. Please also list any relevant details for your test configuration
+
+- [ ] Your changes have to be tested on hardware
+- [ ] You have added unit tests
+
+## Test A
+1) Step 1
+2) Step 2
+
+## Test B
+1) Step 1
+2) Step 2
+
+## Roslint Test
+
+- [ ] You have performed linter tests
```

Comment:
@patapufate I'd suggest we move this to the checklist. A `roslint` test should be a must.

Reply:
Yes. You're right